tests/e2e/testdata/scenarios/direct/logging/acquire/_http01.log

```diff
@@ -0,0 +1,199 @@
+Server: ESF
+Server-Timing: gfet4t7; dur=60
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "createTime": "2024-04-01T12:34:56.123456Z",
+  "filter": "resource.type=gae_app AND severity\u003c=ERROR",
+  "metricDescriptor": {
+    "metricKind": "DELTA",
+    "name": "projects/${projectId}/metricDescriptors/logging.googleapis.com/user/logginglogmetric-to-acquire",
+    "type": "logging.googleapis.com/user/logginglogmetric-to-acquire",
+    "unit": "1",
+    "valueType": "INT64"
+  },
+  "name": "logginglogmetric-to-acquire",
+  "updateTime": "2024-04-01T12:34:56.123456Z"
+}
+
+---
+
+GET https://logging.googleapis.com/v2/projects/${projectId}/metrics/logginglogmetric-to-acquire?alt=json
+Content-Type: application/json
+User-Agent: kcc/controller-manager DeclarativeClientLib/0.0.1
+
+200 OK
+Cache-Control: private
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Server-Timing: gfet4t7; dur=46
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "createTime": "2024-04-01T12:34:56.123456Z",
+  "filter": "resource.type=gae_app AND severity\u003c=ERROR",
+  "metricDescriptor": {
+    "metricKind": "DELTA",
+    "name": "projects/${projectId}/metricDescriptors/logging.googleapis.com/user/logginglogmetric-to-acquire",
+    "type": "logging.googleapis.com/user/logginglogmetric-to-acquire",
+    "unit": "1",
+    "valueType": "INT64"
+  },
+  "name": "logginglogmetric-to-acquire",
+  "updateTime": "2024-04-01T12:34:56.123456Z"
+}
+
+---
+
+GET https://logging.googleapis.com/v2/projects/${projectId}/metrics/logginglogmetric-to-acquire?alt=json
+Content-Type: application/json
+User-Agent: kcc/controller-manager DeclarativeClientLib/0.0.1
+
+200 OK
+Cache-Control: private
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Server-Timing: gfet4t7; dur=56
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "createTime": "2024-04-01T12:34:56.123456Z",
+  "filter": "resource.type=gae_app AND severity\u003c=ERROR",
+  "metricDescriptor": {
+    "metricKind": "DELTA",
+    "name": "projects/${projectId}/metricDescriptors/logging.googleapis.com/user/logginglogmetric-to-acquire",
+    "type": "logging.googleapis.com/user/logginglogmetric-to-acquire",
+    "unit": "1",
+    "valueType": "INT64"
+  },
+  "name": "logginglogmetric-to-acquire",
+  "updateTime": "2024-04-01T12:34:56.123456Z"
+}
+
+---
+
+PUT https://logging.googleapis.com/v2/projects/${projectId}/metrics/logginglogmetric-to-acquire?alt=json
+Content-Type: application/json
+User-Agent: kcc/controller-manager DeclarativeClientLib/0.0.1
+
+{
+  "filter": "resource.type=gae_app AND severity\u003c=INFO",
+  "metricDescriptor": {
+    "labels": [],
+    "metricKind": "DELTA",
+    "unit": "1",
+    "valueType": "INT64"
+  }
+}
```

Comment:
surprised to see this PUT from the dcl controller while acquiring. Curious what the diff computed was that required an upgrade, it looks like it's adding the `labels` slice at first sight.

Reply:
Yeah, maybe we need the PR that logs the diff in the controller to be sure.